lib/cdo/i18n_backend.rb

```diff
@@ -140,16 +141,29 @@ def translate(locale, key, options = ::I18n::EMPTY_HASH)
       end
     end
 
+    # Plugin for logging usage information about i18n strings.
+    module I18nStringUrlTrackerPlugin
+      def translate(locale, key, options = ::I18n::EMPTY_HASH)
+        result = super
+        url = Thread.current[:current_request_url]
+        normalized_key = key.to_s.downcase
```

Comment:
We also need to be able to support calls that use the `scope` parameter; see https://www.rubydoc.info/github/svenfuchs/i18n/master/I18n%2FBase:translate for more info, and see the `SmartTranslate` module in this file for an example of other code that interacts with `scope` in an effort to normalize

Reply:
Oh, we could possibly just use the existing `normalize_keys` functionality:

https://github.com/ruby-i18n/i18n/blob/c2d85c0bf61c27e1a732c16eebccc1999c6cbaf4/lib/i18n.rb#L320-L331

Reply:
fixed